frontend/src/scenes/onboarding/sdks/SDKs.tsx

```diff
@@ -1,54 +1,74 @@
+}): JSX.Element {
+    const { setSourceFilter, setSelectedSDK, setAvailableSDKInstructionsMap } = useActions(sdksLogic)
     const { sourceFilter, sdks, selectedSDK, sourceOptions } = useValues(sdksLogic)
     const { productKey } = useValues(onboardingLogic)
 
+    useEffect(() => {
+        setAvailableSDKInstructionsMap(sdkInstructionMap)
+    }, [])
+
     return (
         <OnboardingStep
             title={`Where are you ${usersAction || 'collecting data'} from?`}
-            subtitle="Pick one to start and add more sources later."
+            subtitle="Pick one or two to start and add more sources later."
         >
-            <div className="flex gap-x-4">
-                <LemonInput placeholder="Search for a source" type="search" />
-                {sourceOptions.length > 1 && (
-                    <LemonSelect
-                        allowClear
-                        onChange={(v) => setSourceFilter(v)}
-                        options={sourceOptions}
-                        placeholder="Select a source type"
-                        value={sourceFilter}
-                    />
+            <LemonDivider className="my-8" />
+            <div className="flex gap-x-8 mt-8">
+                <div className={`flex flex-col gap-y-2 flex-wrap gap-x-4 min-w-50`}>
+                    {sourceOptions.length > 1 && (
+                        <LemonSelect
+                            allowClear
+                            onChange={(v) => setSourceFilter(v)}
+                            options={sourceOptions}
+                            placeholder="Select a source type"
+                            value={sourceFilter}
+                            fullWidth
+                        />
+                    )}
+                    {sdks?.map((sdk) => (
+                        <React.Fragment key={`sdk-${sdk.key}`}>
+                            <LemonButton
+                                status={selectedSDK?.key === sdk.key ? 'primary' : 'muted-alt'}
+                                active={selectedSDK?.key === sdk.key}
+                                onClick={selectedSDK?.key !== sdk.key ? () => setSelectedSDK(sdk) : undefined}
+                                fullWidth
+                                icon={
+                                    <div className="w-4">
+                                        {typeof sdk.image === 'string' ? (
+                                            <img src={sdk.image} className="w-4" />
+                                        ) : (
+                                            sdk.image
+                                        )}
+                                    </div>
+                                }
+                            >
+                                {sdk.name}
+                            </LemonButton>
+                        </React.Fragment>
+                    ))}
+                </div>
+                {selectedSDK && productKey && (
```

Comment:
Will this work? I think you can't have the double `&&`. Something like this should work instead:
```
{selectedSDK && productKey ? (
    ...
) : null}

Reply:
&& works :) Sometimes the `? :` syntax is more explicit but it feels unnecessary sometimes.

Reply:
TIL